### PR TITLE
s[e-n]*: normalize docs

### PR DESCRIPTION
**Pull request by russoz**
##### SUMMARY

Normalize doc blocks.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

plugins/modules/sefcontext.py
plugins/modules/selinux_permissive.py
plugins/modules/selogin.py
plugins/modules/sendgrid.py
plugins/modules/sensu_check.py
plugins/modules/sensu_client.py
plugins/modules/sensu_handler.py
plugins/modules/sensu_silence.py
plugins/modules/sensu_subscription.py
plugins/modules/seport.py
plugins/modules/serverless.py
plugins/modules/shutdown.py
plugins/modules/simpleinit_msb.py
plugins/modules/sl_vm.py
plugins/modules/slack.py
plugins/modules/slackpkg.py
plugins/modules/smartos_image_info.py
plugins/modules/snap.py
plugins/modules/snap_alias.py
plugins/modules/snmp_facts.py

### CODE REVIEW

File: plugins/modules/sefcontext.py [hunk 1]
```diff
@@ -28,85 +27,82 @@
...
+      - The file type that should have SELinux contexts applied.
+      - The following file type options are available:.
+      - V(a) for all files,.
+      - V(b) for block devices,.
+      - V(c) for character devices,.
+      - V(d) for directories,.
+      - V(f) for regular files,.
+      - V(l) for symbolic links,.
+      - V(p) for named pipes,.
+      - V(s) for socket files.
```
Comment (felixfontein) [inline plugins/modules/sefcontext.py:45]: A trailing comma should also be left alone:
```suggestion
      - The following file type options are available:
      - V(a) for all files,
      - V(b) for block devices,
      - V(c) for character devices,
      - V(d) for directories,
      - V(f) for regular files,
      - V(l) for symbolic links,
      - V(p) for named pipes,
      - V(s) for socket files.
```

File: plugins/modules/sensu_check.py [hunk 2]
```diff
@@ -29,127 +28,127 @@
...
       - Will be created if it does not exist (unless O(state=absent)).
-      - The parent folders need to exist when O(state=present), otherwise an error will be thrown
+      - The parent folders need to exist when O(state=present), otherwise an error will be thrown.
     default: /etc/sensu/conf.d/checks.json
   backup:
     description:
-      - Create a backup file (if yes), including the timestamp information so
-      - you can get the original file back if you somehow clobbered it incorrectly.
+      - Create a backup file (if yes), including the timestamp information so.
+      - You can get the original file back if you somehow clobbered it incorrectly.
```
Comment (felixfontein) [inline plugins/modules/sensu_check.py:50]:
```suggestion
      - Create a backup file (if yes), including the timestamp information so
        you can get the original file back if you somehow clobbered it incorrectly.
```

File: plugins/modules/sensu_check.py [hunk 3]
```diff
@@ -29,127 +28,127 @@
...
-      - This option obviates the need for specifying the O(subscribers) option
+      - Whether the check should be scheduled by the sensu client or server.
+      - This option obviates the need for specifying the O(subscribers) option.
       - Default is V(false).
     type: bool
   publish:
     description:
       - Whether the check should be scheduled at all.
-      - You can still issue it via the sensu api
+      - You can still issue it via the sensu api.
```
Comment (felixfontein) [inline plugins/modules/sensu_check.py:113]:
```suggestion
      - You can still issue it via the sensu API.
```

File: plugins/modules/sensu_check.py [hunk 4]
```diff
@@ -29,127 +28,127 @@
...
     type: int
     description:
-      - Number of seconds handlers should wait before taking second action
+      - Number of seconds handlers should wait before taking second action.
   aggregate:
     description:
-      - Classifies the check as an aggregate check,
-      - making it available via the aggregate API
+      - Classifies the check as an aggregate check,.
+      - Making it available via the aggregate API.
```
Comment (felixfontein) [inline plugins/modules/sensu_check.py:128]:
```suggestion
      - Classifies the check as an aggregate check,
        making it available via the aggregate API.
```

File: plugins/modules/sensu_client.py [hunk 5]
```diff
@@ -8,14 +8,13 @@
...
-DOCUMENTATION = '''
----
+DOCUMENTATION = r"""
 module: sensu_client
 author: "David Moreau Simard (@dmsimard)"
 short_description: Manages Sensu client configuration
 description:
   - Manages Sensu client configuration.
-  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/clients.html)'
+  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/clients.html).'
```
Comment (felixfontein) [inline plugins/modules/sensu_client.py:17]:
```suggestion
  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/clients.html)'
```

File: plugins/modules/sensu_client.py [hunk 6]
```diff
@@ -100,10 +100,10 @@
     description:
       - The servicenow definition scope, used to configure the Sensu Enterprise ServiceNow integration (Sensu Enterprise users only).
 notes:
-  - Check mode is supported
-'''
+  - Check mode is supported.
```
Comment (felixfontein) [inline plugins/modules/sensu_client.py:103]: This doesn't work with suggestions, but can you remove the whole `notes:`? The only note is already covered by `attributes`.

File: plugins/modules/sensu_handler.py [hunk 7]
```diff
@@ -8,14 +8,13 @@
...
----
+DOCUMENTATION = r"""
 module: sensu_handler
 author: "David Moreau Simard (@dmsimard)"
 short_description: Manages Sensu handler configuration
 description:
-  - Manages Sensu handler configuration
-  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/handlers.html)'
+  - Manages Sensu handler configuration.
+  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/handlers.html).'
```
Comment (felixfontein) [inline plugins/modules/sensu_handler.py:17]:
```suggestion
  - 'For more information, refer to the Sensu documentation: U(https://sensuapp.org/docs/latest/reference/handlers.html)'
```

File: plugins/modules/sensu_handler.py [hunk 8]
```diff
@@ -100,10 +99,10 @@
       - An array of Sensu event handlers (names) to use for events using the handler set.
       - 'NOTE: the handlers attribute is only required for handler sets (i.e. handlers configured with "type": "set").'
```
Comment (felixfontein) [inline plugins/modules/sensu_handler.py:100]:
```suggestion
      - 'NOTE: the handlers attribute is only required for handler sets (that is, handlers configured with O(type=set)).'
```

File: plugins/modules/sensu_subscription.py [hunk 9]
```diff
@@ -26,49 +25,49 @@
...
-      - Path to the subscriptions json file
+      - Path to the subscriptions json file.
     required: false
     default: /etc/sensu/conf.d/subscriptions.json
   backup:
     description:
-      - Create a backup file (if yes), including the timestamp information so you
-      - can get the original file back if you somehow clobbered it incorrectly.
+      - Create a backup file (if yes), including the timestamp information so you.
+      - Can get the original file back if you somehow clobbered it incorrectly.
```
Comment (felixfontein) [inline plugins/modules/sensu_subscription.py:46]:
```suggestion
      - Create a backup file (if yes), including the timestamp information so you
        can get the original file back if you somehow clobbered it incorrectly.
```

File: plugins/modules/serverless.py [hunk 10]
```diff
@@ -67,16 +66,15 @@
...
     default: false
 notes:
-   - Currently, the C(serverless) command must be in the path of the node executing the task.
-     In the future this may be a flag.
+  - Currently, the C(serverless) command must be in the path of the node executing the task. In the future this may be a flag.
 requirements:
-- serverless
-- yaml
+  - serverless
+  - yaml
```
Comment (felixfontein) [inline plugins/modules/serverless.py:72]:
```suggestion
  - PyYAML
```

File: plugins/modules/slack.py [hunk 11]
```diff
@@ -32,51 +32,42 @@
...
+      - Message to send. Note that the module does not handle escaping characters. Plain-text angle brackets and ampersands should be converted
+        to HTML entities (for example C(&) to C(&amp;)) before sending. See Slack's documentation (U(https://api.slack.com/docs/message-formatting)) for more.
   channel:
     type: str
     description:
       - Channel to send the message to. If absent, the message goes to the channel selected for the O(token).
   thread_id:
     description:
-      - Optional. Timestamp of parent message to thread this message. https://api.slack.com/docs/message-threading
+      - Optional. Timestamp of parent message to thread this message. https://api.slack.com/docs/message-threading.
```
Comment (felixfontein) [inline plugins/modules/slack.py:64]:
```suggestion
      - Optional. Timestamp of parent message to thread this message, see U(https://api.slack.com/docs/message-threading).
```

File: plugins/modules/snmp_facts.py [hunk 12]
```diff
@@ -9,87 +9,85 @@
...
+DOCUMENTATION = r"""
 module: snmp_facts
 author:
-- Patrick Ogenstad (@ogenstad)
+  - Patrick Ogenstad (@ogenstad)
 short_description: Retrieve facts for a device using SNMP
 description:
-    - Retrieve facts for a device using SNMP, the facts will be
-      inserted to the ansible_facts key.
+  - Retrieve facts for a device using SNMP, the facts will be inserted to the ansible_facts key.
```
Comment (felixfontein) [inline plugins/modules/snmp_facts.py:18]:
```suggestion
  - Retrieve facts for a device using SNMP, the facts will be inserted to the C(ansible_facts) key.
```

File: plugins/modules/sensu_client.py (hunk 5 above)
Comment (russoz) [reply plugins/modules/sensu_client.py:17]: Accepting this now, but I raised an issue to move these links to the seealso section.

File: plugins/modules/sensu_client.py (hunk 6 above)
Comment (russoz) [reply plugins/modules/sensu_client.py:103]: Will do this on the next round of review, after the batch commit

File: plugins/modules/sensu_handler.py [hunk 13]
```diff
@@ -100,10 +99,10 @@
...
 notes:
-  - Check mode is supported
-'''
+  - Check mode is supported.
```
Comment (russoz) [inline plugins/modules/sensu_handler.py:102]: Note to self: remove

File: plugins/modules/slack.py (hunk 11 above)
Comment (russoz) [reply plugins/modules/slack.py:64]: Should we remove the period after the URL at the end of the line as well? You requested this in this in `sensu_handler` (and maybe another ones) in this very PR.

Comment (russoz) [reply plugins/modules/slack.py:64]: Not adding this to the batch this time around.

File: plugins/modules/smartos_image_info.py [hunk 14]
```diff
@@ -9,31 +9,28 @@
...
-              manifest and C(published_date), C(published), C(source), C(clones),
-              and C(size). More information can be found at U(https://smartos.org/man/1m/imgadm)
-              under C(imgadm list).
-        type: str
-'''
-
-EXAMPLES = '''
+  filters:
+    description:
+      - Criteria for selecting image. Can be any value from image manifest and C(published_date), C(published), C(source), C(clones), and C(size).
```
Comment (russoz) [inline plugins/modules/smartos_image_info.py:28]:
```suggestion
      - Criteria for selecting image. Can be any value from image manifest and V(published_date), V(published), V(source), V(clones), and V(size).
```

File: plugins/modules/slack.py (hunk 11 above)
Comment (felixfontein) [reply plugins/modules/slack.py:64]: No, the period should stay; in the sensu_handler module it was `sentence: URL` (no period needed) and not `Sentence, see URL.` (period needed). Simply having `Sentence. URL` is strange with or without period ;)

File: plugins/modules/sensu_client.py (hunk 6 above)
Comment (russoz) [reply plugins/modules/sensu_client.py]: done

File: plugins/modules/seport.py [hunk 15]
```diff
@@ -53,26 +52,26 @@
...
-    - Work with local modifications only.
+      - Work with local modifications only.
     type: bool
     default: false
     version_added: 5.6.0
 notes:
-   - The changes are persistent across reboots.
-   - Not tested on any debian based system.
+  - The changes are persistent across reboots.
+  - Not tested on any debian based system.
```
Comment (felixfontein) [inline plugins/modules/seport.py:66]:
```suggestion
  - Not tested on any Debian based system.
```

**Review by felixfontein — COMMENTED** [review]
Sorry, one more :)

**Review by felixfontein — APPROVED** [review]

**Comment by felixfontein** [comment]
@russoz thanks!

**Pull request merged**